podcast.rss

```diff
@@ -7341,5 +7341,349 @@
+        <![CDATA[<p>Velicham Qur'an Dars series</p><p>Qur'an class #637 💠Surah An-Naba 💠 *ആയത്ത്&zwnj; -12* وَبَنَيْنَا فَوْقَكُمْ سَبْعًۭا شِدَادًۭا നിങ്ങളുടെ മേലെ നാം ശക്തമായ ഏഴെണ്ണം നിർമ്മിച്ചു. 🔸Admin Team Velicham Onlive🔸</p>]]>
+      </description>
+      <enclosure url="https://velicham-onlive.github.io/audios/78-naba/12-vqd-podcast-unit78.mp3" type="audio/mp3" length="2471346"/>
+      <itunes:duration>11:44</itunes:duration>
+      <pubDate>Tuesday, 10 November ‎2020 ‏‎12:00:00 GMT</pubDate>
+      <guid>12-vqd-podcast-unit78</guid>
+    </item>
+    <!--  -->
+     <item>
+      <title>638.സൂറഃ നബ ആയത്ത് 13_a</title>
+      <description>
+        <![CDATA[<p>Velicham Qur'an Dars series</p><p>Qur'an class #638 💠Surah An-Naba 💠 *ആയത്ത്&zwnj; -13* وَجَعَلْنَا سِرَاجًۭا وَهَّاجًۭا കത്തിജ്ജ്വലിക്കുന്ന ഒരു വിളക്ക്&zwnj; നാം ഉണ്ടാക്കി. 🔸Admin Team Velicham Onlive🔸</p>]]>
+      </description>
+      <enclosure url="https://velicham-onlive.github.io/audios/78-naba/13-vqd-podcast-unit78.mp3" type="audio/mp3" length="2162033"/>
+      <itunes:duration>10:03</itunes:duration>
+      <pubDate>Tuesday, 10 November ‎2020 ‏‎12:00:00 GMT</pubDate>
+      <guid>13-vqd-podcast-unit78</guid>
+    </item>
+    <!--  -->
+     <item>
+      <title>639.സൂറഃ നബ ആയത്ത്  14-16 Part 1_a</title>
+      <description>
+        <![CDATA[<p>Velicham Qur'an Dars series</p><p>Qur'an class #639 💠Surah An-Naba 💠 *ആയത്ത്&zwnj; -14,15&amp;16* വിവരണം -1 وَأَنزَلْنَا مِنَ ٱلْمُعْصِرَٰتِ مَآءًۭ ثَجَّاجًۭا മേഘങ്ങളിൽനിന്ന് നാം കുത്തിയൊഴുകുന്ന വെള്ളം ഇറക്കുകയും ചെയ്തു. لِّنُخْرِجَ بِهِۦ حَبًّۭا وَنَبَاتًۭا അതുവഴി ധാന്യവും ചെടിയും ഉല്&zwj;പാദിപ്പിക്കാന്&zwj;. وَجَنَّٰتٍ أَلْفَافًا ഇടതൂര്&zwj;ന്ന തോട്ടങ്ങളും. 🔸Admin Team Velicham Onlive🔸</p>]]>
+      </description>
+      <enclosure url="https://velicham-onlive.github.io/audios/78-naba/14-vqd-podcast-unit78.mp3" type="audio/mp3" length="2423825"/>
+      <itunes:duration>11:18</itunes:duration>
+      <pubDate>Tuesday, 10 November ‎2020 ‏‎12:00:00 GMT</pubDate>
+      <guid>14-vqd-podcast-unit78</guid>
+    </item>
+    <!--  -->
+    <item>
+      <title>640.സൂറഃ നബ ആയത്ത് 14-16 Part 2_a</title>
+      <description>
+        <![CDATA[<p>Velicham Qur'an Dars series</p><p>Qur'an class #640 💠Surah An-Naba 💠 *ആയത്ത്&zwnj; -14,15&amp;16* വിവരണം -2 وَأَنزَلْنَا مِنَ ٱلْمُعْصِرَٰتِ مَآءًۭ ثَجَّاجًۭا മേഘങ്ങളിൽനിന്ന് നാം കുത്തിയൊഴുകുന്ന വെള്ളം ഇറക്കുകയും ചെയ്തു. لِّنُخْرِجَ بِهِۦ حَبًّۭا وَنَبَاتًۭا അതുവഴി ധാന്യവും ചെടിയും ഉല്&zwj;പാദിപ്പിക്കാന്&zwj;. وَجَنَّٰتٍ أَلْفَافًا ഇടതൂര്&zwj;ന്ന തോട്ടങ്ങളും. 🔸Admin Team Velicham Onlive🔸</p>]]>
+      </description>
+      <enclosure url="https://velicham-onlive.github.io/audios/78-naba/15-vqd-podcast-unit78.mp3" type="audio/mp3" length="2712329"/>
+      <itunes:duration>12:34</itunes:duration>
+      <pubDate>Tuesday, 10 November ‎2020 ‏‎12:00:00 GMT</pubDate>
+      <guid>15-vqd-podcast-unit78</guid>
+    </item>
+    <!--  -->
+    <item>
+      <title>641.സൂറഃ നബഅ്   ആയത്(17)_a</title>
```

Comment:
@febincp6 let's keep a uniform format for title. Can we remove () from all titles?